app/models.py

```diff
@@ -118,9 +118,11 @@ def commit(self):
         super().commit()
 
 
-VerifiedSession = scoped_session(
-    sessionmaker(class_=_VerifiedSession), scopefunc=session_context_id.get
-)
+def VerifiedSession(user_or_token):
```

Comment:
Ah, interesting. The session_maker doesn't do this by default?

Reply:
Doesn't do what by default?